### PR TITLE
Update CSI documentation to 4.0

### PR DESCRIPTION
**Pull request by jsafrane**
Note to self: re-test when https://github.com/openshift/csi-driver-registrar/pull/5 is available in 4.0 images.

### CODE REVIEW

File: install_config/persistent_storage/persistent_storage_csi.adoc [hunk 1]
```diff
@@ -180,7 +180,7 @@ spec:
       serviceAccount: cinder-csi
       containers:
         - name: csi-attacher
-          image: registry.redhat.io/openshift3/csi-attacher:v3.11
+          image: registry.redhat.io/openshift3/csi-attacher:v4.0
```
Comment (liangxia) [inline install_config/persistent_storage/persistent_storage_csi.adoc]: Are we still using openshift3 ?

Comment (jsafrane) [reply install_config/persistent_storage/persistent_storage_csi.adoc]: Good catch, fixed

**Comment by vikram-redhat** [comment]
@huffmanca just making sure that this is on your radar.

**Comment by liangxia** [comment]
@jsafrane Given we have bug https://bugzilla.redhat.com/show_bug.cgi?id=1665799, could you confirm whether the pr need further change ?

**Comment by jsafrane** [comment]
in bug 1665799 we disabled CSI operator, which means that CSI drivers must  be installed "the old way", i.e. as described in this PR.

**Comment by huffmanca** [comment]
@jsafrane, The master branch is still tracking the 3.x stream. I've created https://github.com/openshift/openshift-docs/pull/14354 to include and update this section for the 4.x docs. Closing this PR, as further discussion should occur in PR 14354.

**Pull request closed without merging**